scripts/check_tts_health.py

```
@@ -0,0 +1,212 @@
+def check_docker_services() -> None:
+    """Check if Docker services are running."""
+    print(f"\n{'=' * 60}")
+    print("Checking Docker Services")
+    print(f"{'=' * 60}")
+
+    # Try to detect if Docker containers are running
+    import subprocess
+
+    try:
+        result = subprocess.run(
+            ["docker", "ps", "--format", "{{.Names}}"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)